src/markdown-pages/install-with-kurl/system-requirements.md

```diff
@@ -16,43 +16,70 @@ title: "System Requirements"
 * Oracle Linux 7.4<sup>\*</sup>, 7.5<sup>\*</sup>, 7.6<sup>\*</sup>, 7.7<sup>\*</sup>, 7.8<sup>\*</sup>, 7.9, 8.0<sup>\*</sup>, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 8.7 (OL 8.x requires Containerd)
 * Amazon Linux 2
 
-*: This version is deprecated since it is no longer supported by its creator. We continue to support it, but support will be removed in the future.
+*&ast; This version is deprecated since it is no longer supported by its creator. We continue to support it, but support will be removed in the future.*
 
 ## Minimum System Requirements
 
 * 4 AMD64 CPUs or equivalent per machine
 * 8 GB of RAM per machine
-* 40 GB of Disk Space per machine
-* The Rook add-on version 1.4.3 and later requires a dedicated block device on each node in the cluster.
-  For more information about how to enable block storage for Rook, see [Block Storage](/docs/add-ons/rook#block-storage) in _Rook Add-On_.
-* TCP ports 2379, 2380, 6443, 10250, 10251 and 10252 open between cluster nodes
-    * **Note**: When [Flannel](/docs/add-ons/flannel) is enabled, UDP port 8472 open between cluster nodes
-    * **Note**: When [Weave](/docs/add-ons/weave) is enabled, TCP port 6783 and UDP port 6783 and 6784 open between cluster nodes
+* 100 GB of Disk Space per machine
+  *(For more specific requirements see [Disk Space Requirements](#disk-space-requirements) below)*
+* TCP ports 2379, 2380, 6443, 10250, 10251 and 10252 and UDP port 8472 (Flannel VXLAN) open between cluster nodes
```

Comment:
Ports `10251` and `10252` have been replaced since Kube 1.17:
> Kubeadm: enable the usage of the secure kube-scheduler and kube-controller-manager ports for health checks. For kube-scheduler was 10251, becomes 10259. For kube-controller-manager was 10252, becomes 10257. (https://github.com/kubernetes/kubernetes/pull/85043, [@neolit123](https://github.com/neolit123))

Source: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#cluster-lifecycle-1

Reply:
We should add these updated ports to the _Network Requirements_ section as well.